src/etc/inc/upgrade_config.inc

```diff
@@ -5493,6 +5493,90 @@ function upgrade_172_to_173() {
 	}
 }
 
+function upgrade_173_to_174() {
```

Comment:
This will need to change, it's already at 17.4 now, so this would be `upgrade_174_to_175()`.